telegram/ext/conversationhandler.py

```diff
@@ -467,13 +467,19 @@ def _schedule_job(
         conversation_key: Tuple[int, ...],
     ) -> None:
         if new_state != self.END:
-            self.timeout_jobs[
-                conversation_key
-            ] = dispatcher.job_queue.run_once(  # type:ignore[union-attr]
-                self._trigger_timeout,  # type: ignore[arg-type]
-                self.conversation_timeout,  # type: ignore[arg-type]
-                context=_ConversationTimeoutContext(conversation_key, update, dispatcher, context),
-            )
+            try:
+                # both job_queue & conversation_timeout are checked before calling _schedule_job
+                j_queue = dispatcher.job_queue
+                self.timeout_jobs[conversation_key] = j_queue.run_once(  # type: ignore[union-attr]
+                    self._trigger_timeout,  # type: ignore[arg-type]
```

Comment:
I think the signature of `_trigger_timeout` is acutally wrong, the argument should be of type `CallbackContext` and this comment can be removed

Reply:
It raises error
```error: "CallbackContext" has no attribute "conversation_key"  [attr-defined]```

Reply:
mh, that's probably because of the unfortunate naming in here:

https://github.com/python-telegram-bot/python-telegram-bot/blob/ccedd3a87d5d7c4ea2a651eba4e9a8bcfba5faba/telegram/ext/conversationhandler.py#L608

Changing that e.g. `cntxt = … ` should probably do the trick. You can tell that in #1920 I didn't put too much effort in getting everything perfect on the first try xD